### PR TITLE
Oregon Working Family Household and Dependent Care Credit 

### PR DESCRIPTION
**Pull request by jingyi0808**
Fixes https://github.com/PolicyEngine/policyengine-us/issues/2863

Oregon Working Family Household and Dependent Care Credit for 2021/2022

### CODE REVIEW

**Comment by MaxGhenis** [comment]
How does this compare to #2969? Please title the PR descriptively and link the issue.

**Comment by PavelMakarchuk** [comment]
please rename the PR to the name of the program

**Comment by jingyi0808** [comment]
Fixes https://github.com/PolicyEngine/policyengine-us/issues/2902

Summary
📝📄✅

This pull request adds a new feature to the policy engine: the Oregon Working Family Household and Dependent Care Credit (WFHDC) It introduces some new parameter files, and adding several new tests cases for the WFHDC test file, as well as updating the changelog entry. The new files define the income threshold for each household size for the WFHDC.

File: policyengine_us/variables/gov/states/or/tax/income/credits/wfhdc/or_wfhdc_table_column.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    entity = TaxUnit
+    label = "Oregon WFHDC percentage table column"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "or_wfhdc_eligible"
+    reference = "https://www.oregon.gov/dor/forms/FormsPubs/publication-or-wfhdc-tb_101-458_2022.pdf"
+    default_value = ""
+
+    def formula(tax_unit, period, parameters):
+        # Column determined by age/disability of youngest child.
```
Comment (leogoldman) [inline policyengine_us/variables/gov/states/or/tax/income/credits/wfhdc/or_wfhdc_table_column.py:15]:
```suggestion
        # Column determined by age of youngest child and whether they have a disability.
```

File: policyengine_us/parameters/gov/states/or/tax/income/credits/wfhdc/table_letter/household_size_8.yaml
Comment (leogoldman) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/wfhdc/table_letter/household_size_8.yaml:1]: Can we rename this to `household_size_8_or_more.yaml` and then update references to it in the variables

File: policyengine_us/variables/gov/states/or/tax/income/credits/wfhdc_householdsize_2.py
Comment (leogoldman) [inline policyengine_us/variables/gov/states/or/tax/income/credits/wfhdc_householdsize_2.py:1]: Please delete this file

File: policyengine_us/tests/policy/baseline/gov/states/or/tax/income/credits/wfhdc/or_wfhdc.yaml
[the remark this answers is not in the document]
Comment (jingyi0808) [reply policyengine_us/tests/policy/baseline/gov/states/or/tax/income/credits/wfhdc/or_wfhdc.yaml:1]: I tried this:

- - name: WFHDC test
  period: 2021
  absolute_error_margin: 0
  input:
    tax_units:
      tax_unit:
        or_wfhdc_table_letter: "A"
        or_wfhdc_table_column: "age_3_to_6"
        cdcc_relevant_expenses: 100
  output:
    or_wfhdc: 8

but it raises an error:"    Could not parse situation described: {'people': 'No person found. At least one person must be defined to run a simulation.'}".

Does this mean that we still have to include people as input?

**Comment by PavelMakarchuk** [comment]
Duplicate of #3403

**Pull request closed without merging**